scripts/solve_network.py

```diff
@@ -1515,6 +1552,11 @@ def extra_functionality(
             logger.info(f"Setting annual volume matching of {energy_matching}%")
             res_annual_matching_constraints(n)
             excess_constraints(n)
+        elif strategy == "emi-match":
+            logger.info(f"Setting annual avoided emission target of {emission_matching}%")
+            emission_matching_constraints(n)
+            #res_annual_matching_constraints(n)
+            #excess_constraints(n)
```

Comment:
```suggestion
        elif strategy == "emi-match":
            logger.info(f"Setting annual avoided emission target of {emission_matching}%")
            logger.info(f"Setting annual volume matching of {energy_matching}%")
            emission_matching_constraints(n)
            res_annual_matching_constraints(n)
            excess_constraints(n)
```

have you test if the model works if `res_annual_matching_constraints(n)` and `excess_constraints(n)` is activated as well?

Reply:
I just tried and it worked.